test/integration/StreamrClient.test.js

```diff
@@ -19,6 +19,14 @@ const createClient = (opts = {}) => new StreamrClient({
 })
 
 describe('StreamrClient Connection', () => {
+    it('can disconnect before connected', async (done) => {
+        const client = createClient()
+        client.once('error', done)
```

Comment:
Two `done`s here now - I guess the behavior we want is that no error is thrown, so remove this line